resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -698,7 +698,7 @@ ecs.execute_command_disable_failed=Execute Command could not be disabled for {0}
 ecs.execute_command_disable_in_progress=Cannot execute action since Execute Command is being disabled for {0}
 ecs.execute_command_disable_progress_indicator_message=Disabling Command Execution for {0}
 ecs.execute_command_disable_success=Execute Command for {0} has been disabled.
-ecs.execute_command_disable_warning=Disabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service.\n Do you wish to continue?
+ecs.execute_command_disable_warning=Disabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service.Do you wish to continue?
```

Comment:
```suggestion
ecs.execute_command_disable_warning=Disabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service. Do you wish to continue?
```

Reply:
This is addressed with a line break in this PR: https://github.com/aws/aws-toolkit-jetbrains/pull/3834